src/constants.ts

```diff
@@ -16,5 +15,5 @@ export const xParserCircularProps = 'x-parser-circular-props';
 
 export const EXTENSION_REGEX = /^x-[\w\d.\-_]+$/;
 
-// Only >=2.0.0 versions are supported
-export const specVersions = Object.keys(specs).filter((version: string) => !['1.0.0', '1.1.0', '1.2.0', '2.0.0-rc1', '2.0.0-rc2'].includes(version));
\ No newline at end of file
+export const specVersions = Object.keys(specs);
```

Comment:
Do we now allow all versions?

Reply:
No, look at this PR https://github.com/asyncapi/spec-json-schemas/pull/248 We need to reduce the size of `specs` package. Now, I will switch to the old code and then I will create new PR in this repo, when we will merge mentioned PR.

Reply:
Ok, I see you changed the code after. 👍